entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+echo "::warning ::This version of cfn-lint-action is deprecated, please upgrade via https://github.com/ScottBrenner/cfn-lint-action#upgrading-from-version-1"
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message